### PR TITLE
update diagnostics for unit/conf changes

### PR DESCRIPTION
**Pull request by sosiouxme**

### CODE REVIEW

**Comment by sosiouxme** [comment]
@sdodson am I missing anything here? For origin are we shipping origin-master and origin-node unit files?

**Comment by sdodson** [comment]
@sosiouxme Yes, origin will have origin-{master,node} and AEP/OSE will have atomic-openshift-{master,node}.

Also, we need to add origin-master-{api,controllers} and atomic-openshift-master-{api,controllers}, this is currently being done via Ansible but we should move those into origin repo. Those services are mutually exclusive with the normal master service though only through ansible work. This hasn't been worked on but we should probably add some sort of meta-service like origin-master-ha that's responsible for starting both of them and conflicts with origin-master.

**Comment by sosiouxme** [comment]
Diagnostics was already neglecting origin units so this doesn't make anything worse. I'll raise a separate issue for that. I will also have to get up to speed on the coming HA changes.

**Comment by sosiouxme** [comment]
@sdodson or @abutcher  would you mind a sanity check on this before I merge? It's for bug https://bugzilla.redhat.com/show_bug.cgi?id=1273757 - Do I need to do anything special to have this cherry-picked to OSE 3.1?

File: pkg/diagnostics/systemd/systemd.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ log message:
   component. Check pod logs and recreate it with the correct CA cert.
   Routers and registries won't work properly with the wrong CA.
 * If it is from a node IP, the client is likely a node. Check the
-  openshift-node logs and reconfigure with the correct CA cert.
+  atomic-openshift-node logs and reconfigure with the correct CA cert.
```
Comment (sdodson) [inline pkg/diagnostics/systemd/systemd.go]: I'm torn as to whether or not we should make this messaging more general and say 'the node logs' or perhaps 'origin-node or atomic-openshift-node logs' ? Not sure it's worth changing from what you've proposed.

**Comment by sdodson** [comment]
LGTM just not sure how generic we want to be in the messaging.

File: pkg/diagnostics/systemd/systemd.go (hunk 1 above)
Comment (sosiouxme) [inline pkg/diagnostics/systemd/systemd.go]: "Node logs" would generalize nicely to be used for both. But I guess I'm inclined to try to be specific for sysadmins who might not immediately mentally map that phrase to specific unit logs.

**Comment by sosiouxme** [comment]
I'd like to address genericizing separately. I'll [merge] this to fix the bug.

**Pull request merged**